docs/products/eigencompute/eigencompute-overview.md

```diff
@@ -3,31 +3,26 @@ title: EigenCompute Overview
 sidebar_position: 4
 ---
 
-## What is EigenCompute?
-EigenCompute is a verifiable compute service enabling you to run offchain workloads in containerized environments with onchain
-verifiability. EigenCompute brings the power of decentralized trust to execution environments traditionally outside the blockchain.
+## Overview
+Build verifiable applications without thinking about determinism, solidity, or how to build a consensus mechanism. Simply containerize your application using docker and upload your application to EigenCompute to ship a verifiable, unstoppable application. 
 
-## Why Use EigenCompute?
-Smart contracts alone aren’t enough for modern apps. Use EigenCompute when your application:
-* Requires AI inference, game simulation, or external data transformations.
-* Needs to offload heavy computation without sacrificing trust.
-* Wants verifiable, onchain-readable results from decentralized operators.
+**Our mission is to enable any developer to build scalable & verifiable applications, so that any developer anywhere in the world can build trusted applications.** 
+We built EigenCompute because we believe that verifiable applications unlock trust for everyone. DeFi lets builders around the world move trillions of dollars onchain without enormous compliance departments because anyone can verify the code. But blockchain verification suffers from scalability and programmability problems. Five cents of Web2 compute can cost over $10K on an L2 and $100K on an L1. This makes verifiable agents, and verifiable high scale applications impossible onchain. 
```

Comment:
> Five cents of Web2 compute can cost over $10K on an L2 and $100K on an L1
Can we put a source. we should not claim something without a source imo in public